src/test/java/chapter5/Question2Test.java

```diff
@@ -0,0 +1,14 @@
+package chapter5;
+
+import org.junit.jupiter.api.Test;
+import src.chapter5.question2.Question2;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Question2Test {
+    @Test
+    public void concatWithX_whenAStringIsValid_shouldReturnAStringReplacedWIthX() {
+        assertTrue(Question2.concatWithX("1234").contains("1234"));
```

Comment:
Use assertEquals with the exact expected string.

Reply:
Fixed.